cookbooks/cdo-varnish/libraries/http_cache.rb

```diff
@@ -197,6 +197,16 @@ def self.config(env)
             headers: [],
             cookies: 'none'
           },
+          {
+            path: '/api/v1/animation-library/*',
+            headers: [],
+            cookies: 'none'
+          },
+          {
+            path: '/api/v1/dataset-library/*',
+            headers: [],
```

Comment:
Not blocking, question for my own learning purposes: what do the headers do for the dataset-library?

Reply:
All HTTP requests have headers, and ignoring all headers here helps us cache these requests better. Also see earlier comments here: https://github.com/code-dot-org/code-dot-org/pull/28411#discussion_r281796324